homework-solutions/ts/task-2.ts

```diff
@@ -0,0 +1,38 @@
+// Создайте функцию validatePassword, которая принимает строку (пароль) и возвращает true,
+// если пароль соответствует следующим правилам:
+//   - Пароль должен содержать хотя бы одну заглавную букву.
+//   - Пароль должен содержать хотя бы одну букву в нижнем регистре.
+//   - Пароль должен содержать хотя бы одну цифру.
+//   - Пароль должен быть не менее 8 символов.
+//   - Пароль не должен состоять из одних пробелов
+// Функция должна возвращать false, если хотя бы одно из условий не выполнено.
+
+enum LETTERS_AND_NUMBERS_OF_PASS {
+  lower = 'abcdefghijklmnopqrstuvwxyz',
+  upper = 'ABCDEFGHIJKLMNOPQRSTUVWXYZ',
+  number = '1234567890',
```

Comment:
Проперти у енамок желательно обозначать в верхнем регистре, при использовании читается приятнее
```suggestion
  LOWER= 'abcdefghijklmnopqrstuvwxyz',
  UPPER= 'ABCDEFGHIJKLMNOPQRSTUVWXYZ',
  NUMBER= '1234567890',
```